gatsby-node.ts

```diff
@@ -105,3 +95,38 @@ export const onCreateNode = async ({
     });
   }
 };
+
+// link geometry and staticMap to metaJson nodes
+export const createSchemaCustomization = ({
+  actions: { createTypes, createFieldExtension },
+}) => {
+  if (process.env.FAST_BUILD === '1') {
```

Comment:
@ohrie ich meine diese `FAST_BUILD` variable, muss noch in netlify eingetragen werden.